docs/reference/ak.behavior.md

```diff
@@ -510,41 +510,46 @@ One may wish to disable this by type-checking, since the functionalities are rat
 ## Adding behavior to arrays
 
 Occasionally, you may want to add behavior to an array that does not contain
-records. A good example of this is to implement strings: strings are not a
-special data type in Awkward Array as they are in many other libraries, they
-are a behavior overlaid on arrays.
+records. Historically, this mechanism was used to help implement strings (although)
+strings have since been made a part of Awkward's internals, alongside categorical types.
 
-There are four predefined string behaviors:
-
-- {class}`ak.CharBehavior`: an array of UTF-8 encoded characters;
-- {class}`ak.ByteBehavior`: an array of unencoded characters;
-- {class}`ak.StringBehavior`: an array of variable-length UTF-8 encoded strings;
-- {class}`ak.ByteStringBehavior`: an array of variable-length unencoded bytestrings.
-
-All four override the string representations (`__str__` and `__repr__`),
-but the string behaviors additionally override equality:
+Awkward Array supports adding behaviors to the list-types in an array. Let's suppose that 
+one wishes to define a special list that defines a `reversed()` method, equivalent to `array[..., ::-1]`.
 
+First, one must define the behavior class
 ```python
->>> ak.Array(["one", "two", "three"]) == ak.Array(["1", "TWO", "three"])
-<Array [False, False, True] type='3 * bool'>
+class ReversibleArray(ak.Array):
+    def reversed(self):
+        index = ak.local_index(self)
+        reversed_index = index[..., ::-1]
+        return self[reversed_index]
```

Comment:
This can also be implemented as

```python
def reversed(self):
    return self[..., ::-1]
```

but the original is fine: it's some computation that one might want to do—it doesn't matter what that computation is.

Reply:
Indeed, I hesitated on adding this. Originally I was going to add a string-related method that inverted the character order. I'm going to change this because it is not an important part of the messaging.

Reply:
```suggestion
    def reversed(self):
        return self[..., ::-1]
```

Reply:
```suggestion
    def reversed(self):
        return self[..., ::-1]
```